README.md

```diff
@@ -1,11 +1,72 @@
 [![Build Status](https://travis-ci.com/SUSE-Enceladus/public-cloud-info-client.svg?branch=master)](https://travis-ci.com/SUSE-Enceladus/public-cloud-info-client)
 
-pint
-====
+# Introduction
 
 pint (Public Cloud INformation Tracker) is a command line client to access
 the REST API provided by the SUSE Public Cloud Information Service. The
 information service provides data about the images SUSE publishes in various
 public cloud frameworks (Amazon, Google, Microsoft) as well as the
 servers that make up the SUSE maintained and operated update infrastructure
 in those frameworks.
+
+## Quick Start
+
+There are various ways to install the Pint Client:
+
+### Install the python package using pip
+
+```bash   
+  $ pip install susepubliccloudinfo
+```
+
+### Install On SUSE Linux using rpm package  
+  
+```bash
+  $ zypper in python3-susepubliccloudinfo
+```
+
+### Install from Source
```

Comment:
I would recommend moving this up to follow the pip install instructions, since the `-e` makes it a developer mode install of the package, i.e. creates symlinks back to the cloned git repo directory... Again this really should be installing under a virtualenv, and not directly into the system Python infrastructure, especially for a developer mode install...

Reply:
yep

Reply:
Potentially suggest two install approaches here:

1. Install latest version normally, i.e. `pip install` without `-e`
2. Install latest sources in developer mode, i.e. `pip install` withe `-e`

The later can be used by a developer for adhoc testing of the development code without having to remember to run a `pip install`.